### DISCUSSION TITLE
Getting stream into TVHE

### DISCUSSION THREAD

**Discussion by jacoats** [streamlink/streamlink#6219] [Q&A] [answered]
### Description

Hello.  New to this. Trying to get streamlink to play a web stream in TVHE.  The website has a plug in  that seems to work. Works in standalone but can get it to stream in TVHE.  The url is https:// ssh101.com/throwbackstream.  See the image for the format but in general it is: pipe/// —streamlink -stdout —default-stream —url https://ssh101.com/live/throwbackstream 410p, best

I get an error message like “no data found “. And the scan fails. Any ideas where I’m going wrong ? Any help would be much appreciated. 

![IMG_7522](https://github.com/user-attachments/assets/fbb3b50b-399a-4cde-bfc9-d2ea42370784)

**Comment by bastimeyer** [comment, marked as answer]
I suggest you read the documentation about `--url` and `--default-stream`. These are only meant for config files. Apart from that, you didn't set a value for `--default-stream` while also trying to set the positional stream-name value.
https://streamlink.github.io/cli.html#command-line-usage

**Comment by malvinas2** [comment]
For integration of Streamlink in TVH see the detailed instructions of Carlos Gomes: 
[https://cgomesu.com/blog/Tvhlink/](https://cgomesu.com/blog/Tvhlink/)
